### PR TITLE
Renaming cfgs and cffs

### PR DESCRIPTION
**Pull request by theofil**
#### PR description:
- renaming cfgs and cffs to conform with naming conventions
- moving 2 cffs from Herwig7Interface/python to Configuration/Generator to be included in validation workflows https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_extendedgen.py
- this PR should be followed by another PR (or amended) adding the DYToll012Jets_5f_NLO_FXFX_Madgraph_Herwig_13TeV_cff.py and DYToll01234Jets_5f_LO_MLM_Madgraph_Herwig_13TeV_cff.py in PyReleaseValidation workflows with appropriate numbering
- once the above is ready, back port to 10_6_X will follow with a new PR

### CODE REVIEW

**Comment by SiewYan** [comment]
please test

**Comment by silviodonato** [comment]
please abort

**Comment by silviodonato** [comment]
please test

**Comment by theofil** [comment]
@agrohsje updated the PR with the 2 new names we discussed for the amcatnlo/madgraph

File: GeneratorInterface/Herwig7Interface/test/DYToLL01234Jets_5FS_TuneCH3_13TeV_madgraphMLM_herwig7_cfg.py [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+        'set /Herwig/Shower/PartnerFinder:ScaleChoice Partner', 
+        'set /Herwig/Shower/KinematicsReconstructor:InitialInitialBoostOption LongTransBoost', 
+        'set /Herwig/Shower/KinematicsReconstructor:ReconstructionOption General', 
+        'set /Herwig/Shower/KinematicsReconstructor:InitialStateReconOption Rapidity', 
+        'set /Herwig/Shower/ShowerHandler:SpinCorrelations Yes', 
+        'cd /Herwig/Shower', 
+        'set /Herwig/Shower/FxFxHandler:MPIHandler  /Herwig/UnderlyingEvent/MPIHandler', 
+        'set /Herwig/Shower/FxFxHandler:RemDecayer  /Herwig/Partons/RemnantDecayer', 
+        'set /Herwig/Shower/FxFxHandler:ShowerAlpha  AlphaQCD', 
+        'set FxFxHandler:HeavyQVeto Yes',
```
Comment (agrohsje) [inline GeneratorInterface/Herwig7Interface/test/DYToLL01234Jets_5FS_TuneCH3_13TeV_madgraphMLM_herwig7_cfg.py:195]: @Dominic-Stafford : Did you check how to apply the setting?

Comment (Dominic-Stafford) [reply GeneratorInterface/Herwig7Interface/test/DYToLL01234Jets_5FS_TuneCH3_13TeV_madgraphMLM_herwig7_cfg.py:195]: Andreas said that this setting should be set to "Yes" if we want to match the b jets to b quarks, and recommended we do this- apparently this is independent of whether we use the 4fs or the 5fs

**Comment by agrohsje** [comment]
please test

**Comment by agrohsje** [comment]
+generators

**Comment by qliphy** [comment]
+1

**Pull request merged**